.nx/workspace-data/d/daemon.log

```diff
@@ -0,0 +1,14 @@
+node:internal/modules/cjs/loader:1137
+  throw err;
+  ^
+
+Error: Cannot find module '/Users/hynes/Projects/dialtone/node_modules/.pnpm/nx@19.2.3/node_modules/nx/src/daemon/server/start.js'
+    at Module._resolveFilename (node:internal/modules/cjs/loader:1134:15)
+    at Module._load (node:internal/modules/cjs/loader:975:27)
+    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
+    at node:internal/main/run_main_module:28:49 {
+  code: 'MODULE_NOT_FOUND',
+  requireStack: []
+}
+
+Node.js v18.19.0
```

Comment:
Note sure why this got committed, let alone created. I'd have expected it to be ignored at least. Probably can delete it.

Reply:
interesting.. that entire folder should be gitignored. Yeah you can delete